### ISSUE TITLE
Participation coefficient definition

### ISSUE THREAD

**Issue by cassianord** [AlexChristensen/NetworkToolbox#7]
**Summary** <br />
Participation coefficient is not as defined in Guimera & Amaral (2005). 
While in the package it is defined as: 

> "Values closer to 1 suggest greater within-community connectivity and values closer to 0 suggest greater between-community connectivity"

in Guimera & Amaral (2005) it is the other way around: 

> The participation coefficient of a node is therefore close to 1 if its links are uniformly distributed among all the modules and 0 if all its links are within its own module.

<br />

**Reproduce** <br />
Example with all links constrained within the communities:

m <- rbind(
          c(0,1,1,0,0,0,0,0,0,0),
          c(1,0,1,0,0,0,0,0,0,0),
          c(1,1,0,0,0,0,0,0,0,0),
          c(0,0,0,0,1,1,0,0,0,0),
          c(0,0,0,1,0,1,0,0,0,0),
          c(0,0,0,1,1,0,0,0,0,0),
          c(0,0,0,0,0,0,0,1,0,0),
          c(0,0,0,0,0,0,1,0,0,0),
          c(0,0,0,0,0,0,0,0,0,1),
          c(0,0,0,0,0,0,0,0,1,0)
) 

com <- c(1,1,1,2,2,2,3,3,4,4)
participation(m, com)

The result is indeed in line with the package definition, but not as defined by Guimera & Amaral (2005).

<br />

**NetworkToolbox Version** <br />
NetworkToolbox 1.4.2

<br />

**Computer Specifications** <br />
Windows 10 v.21H1

<br />

**Comment by AlexChristensen** [comment]
Hi @cassianord,

Thanks for pointing this inconsistency. It looks like the inconsistency was in the documentation and not the code (at least in the latest 1.4.3 version).

Running your code provided all zeros for the participation coefficient, which is in line with Guimera & Amaral (2005), but not consistent with the documentation.

I verified that having the opposite of your matrix (all between-community connections and no within-community connections) should lead to non-zero, high participation coefficients

```
participation(1 - m, com)
```

The result is as expected with Guimera & Amaral (2005)

```
$overall
 [1] 0.6530612 0.6530612 0.6530612 0.6530612 0.6530612 0.6530612 0.6562500
 [8] 0.6562500 0.6562500 0.6562500
```

This issue might have been different in version 1.4.2 but it seems to have been resolved in 1.4.3. In the latest GitHub push, I changed the documentation to be consistent with the output and Guimera & Amaral (2005)

> Values closer to 1 suggest greater within-community connectivity and values closer to 0 suggest greater between-community connectivity

Please download the latest GitHub version and let me know if any issues persist

```
if(!"devtools" %in% row.names(installed.packages())){
  install.packages("devtools")
}

devtools::install_github("AlexChristensen/NetworkToolbox")
```

**Comment by cassianord** [comment]
Hi, @AlexChristensen !

Checked version 1.4.3 here, the problem is indeed solved.

In 1.4.2 the result of the code above was:

```
> participation(m, com)
$overall
 [1] 1 1 1 1 1 1 1 1 1 1
```

and the opposite of "m" was:

```
> participation(1 - m, com) 
$overall
 [1] 0.3469388 0.3469388 0.3469388 0.3469388 0.3469388 0.3469388 0.3437500 0.3437500 0.3437500 0.3437500
```

Now with 1.4.3, the results are correct. With matrix "m":

```
> participation(m, com)
$overall
 [1] 0 0 0 0 0 0 0 0 0 0
```

With "(1-m)":
```
> participation(1-m, com)
$overall
 [1] 0.6530612 0.6530612 0.6530612 0.6530612 0.6530612 0.6530612 0.6562500 0.6562500 0.6562500 0.6562500
```

Thanks, @AlexChristensen !
